src/Command/Db/DbSizeCommand.php

```diff
@@ -78,19 +82,40 @@ protected function execute(InputInterface $input, OutputInterface $output) {
         $machineReadable = $table->formatIsMachineReadable();
         $showInBytes = $input->getOption('bytes') || $machineReadable;
 
-        $columns  = ['max' => 'Allocated disk', 'used' => 'Estimated usage', 'percent_used' => '% used'];
+        $columns  = ['max' => 'Allocated disk', 'used' => 'Estimated usage', 'percent_used' => 'Percentage used'];
```

Comment:
There are some irrelevant changes in the diff, like this one

Reply:
I removed that change